lib/dal/src/func/binding.rs

```diff
@@ -351,6 +355,51 @@ impl FuncBinding {
         let (context, rx) = FuncDispatchContext::new(ctx);
         Ok((func, execution, context, rx))
     }
+
+    pub async fn execute_dummy(
```

Comment:
Let's let `FuncBinding` own dummy execution since we need its private methods and exposing them is likely not a good idea.